client/src/views/header/NotificationCenterPopover.vue

```diff
@@ -68,12 +67,12 @@ const notifications = computed(() => {
 const onlyReadToggle: Ref<boolean> = ref(false);
 
 onMounted(() => {
-  // Add a new notification every 5 seconds, for a total of 10 notifications
-  for (let i = 1; i < 10; i++) {
+  // WorkspaceRoleChangedNotification
+  for (let i = 1; i < 2; i++) {
```

Comment:
Use a number a bit higher than 2 in order to have more notifications by default.